cookbooks/bcpc-hadoop/attributes/hive.rb

```diff
@@ -1,3 +1,5 @@
-default[:bach][:hive][:metastore][:port] = 9083
-default[:bach][:hive][:hiveserver2][:port] = 10000
-default[:bcpc][:hive][:heap][:size]=1024
+default[:bcpc][:hadoop][:hive][:heap][:size]=1024
+default[:bcpc][:hadoop][:hive][:hiveserver2][:authentication][:method] = "KERBEROS"
+default[:bcpc][:hadoop][:hive][:hiveserver2][:authentication][:ldap_url] = ""
+default[:bcpc][:hadoop][:hive][:hiveserver2][:authentication][:ldap_domain] = ""
```

Comment:
So default hive server auth is kerberos. And kerberos is disabled by default. In that case, by default, no authentication is configured for hive server?